### DISCUSSION TITLE
Support for gpt4o?

### DISCUSSION THREAD

**Discussion by GalShaffirLibba** [RageAgainstThePixel/com.openai.unity#239] [Q&A]
Hello, 
First I just want to show my appreciation for this awesome package, I use it for a while now and it works great. awesome job guys. 
I have a question regarding the support of the plugin for the latest addition to gpt4o:
https://openai.com/index/hello-gpt-4o/

Is it possible now or will be possible in the future to send an audio clip to the model and receive an audio clip from the model?

Thanks!

**Comment by StephenHodgson** [comment]
It's already supported in chat requests. Assistant support coming in #235

**Reply by GalShaffirLibba** [reply]
thanks for you answer stephen,
could you assist me on how to do with chat request? I can't simply add an audio clip to the list of content i'm sending like i do with texutre2d types.

**Reply by StephenHodgson** [reply]
I believe you would need to transcribe it manually in the API first, then just add the text content. I'm unsure if gpt-4o supports adding audio files to conversation directly
